.github/workflows/auto-approve-maintainer-pr.yml

```diff
@@ -16,6 +16,6 @@ jobs:
 
       - name: Auto approve
         uses: hmarr/auto-approve-action@v2
-        if: ${{ steps.is_organization_member.outputs.result == 'true' || github.actor == 'dependabot[bot]' }}
+        if: ${{ github.event.pull_request.draft == false && (steps.is_organization_member.outputs.result == 'true' || github.actor == 'dependabot[bot]') }}
```

Comment:
Can we move this to job level so if it’s a draft this pull request will not run at all?